workflows/postprocessing.nf

```diff
@@ -86,6 +86,25 @@ def exomiser(inputChannel,
     )  
 }
 
+def vep(input_channel, vep_genome, path_fasta, vep_cache, vep_cache_version) {
+    
+    def ch_input_for_vep  = input_channel.map{meta, files ->
+        def vcf_file = files.find { it.name.endsWith("vcf.gz") }
+        def custom_extra_files = [] 
+        [meta, vcf_file, custom_extra_files]
+    }
+  
+    return ENSEMBLVEP_VEP(
+        ch_input_for_vep, 
+        vep_genome,
+        "homo_sapiens", //species
```

Comment:
should we use an arg to define the species.

Reply:
maybe, but I feel it would be unlikely to be used on non-humans, no?

Reply:
Oui c'est pour cette raison que j'avais hard codé le tout. Sans nécéssairement introduire un paramètre, je pourrais utiliser une constante ou juste une variable. Ça clarifierait qu'il s'agit d'une espèce. Je vais voir ce que je peux faire pour améliorer ici.

Reply:
J'ai simplement créé une constante dans le fichier pour l'instant.

Reply:
Hum ... après avoir testé, je vois que ça ne fonctionne pas avec la constante. Je ne sais pas trop pourquoi, mais elle ne semble pas accessible dans la fonction `vep`, sauf si je la mets globale (sans le mot clé def), ce que je n'aime pas trop.

Par contre, elle est accessible dans le corps du workflow.  Si je la passe en argument à la fonction vep à partir du workflow, là ça fonctionne.  C'est ce que j'ai fait pour l'instant. Je vais re-tester sur CQDG pour m'assurer que c'est ok dans un autre contexte d'exécution.